docs/nodes/celestia-node.mdx

```diff
@@ -444,17 +478,31 @@ Error: nodebuilder/share: interval must be positive; nodebuilder/core: invalid I
 Error: nodebuilder/share: interval must be positive
 ```
 
-You can re-initialize your node with the following command:
+You can re-initialize your node with the following commands:
```

Comment:
wait is this only for the config related issue? Check the heading

Reply:
what do you mean? heading is "Clearing data store"

Reply:
https://docs.celestia.org/pr-preview/pr-551/nodes/celestia-node/#clearing-data-store